constants/members.json

```diff
@@ -341,8 +341,8 @@
     },
     {
       "name": "Joyce Shi",
-      "role": "technical lead",
-      "term": 1229,
+      "role": "project lead",
```

Comment:
the role for PL's is actually called `Technical Lead`. This should be changed since it is used for sorting and indexing purposes in `utils.ts`.